### DISCUSSION TITLE
Overflow on Table

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#4692] [Help]
Hey, 

how would I fix the overflow in a Table/Table Card if I have only 1 column?
I wanna get rid of the scrollbar on the right.
Fyi: overflow: hidden; doesn't work.

![Screenshot 2022-02-25 110704](https://user-images.githubusercontent.com/95284278/155696804-2c22c94d-8a0a-43fb-8f26-1763d179fbdd.png)

Thanks!

**Comment by mjashanks** [comment]
Hey @kasgithub8 

There's no easy answer to this, other than inspecting the CSS in dev-tools and trying to work out some selector that might work.

I'd recommend using a `Data Provider` and a `Repeater` (or a `Repeater Block`, which is the same thing).
